src/Extension/FluentExtension.php

```diff
@@ -140,6 +143,14 @@ class FluentExtension extends DataExtension
      */
     private static $frontend_publish_required = true;
 
+    /**
+     * Controls whether the message indicating whether you are editing in the default locale is disabled
+     *
+     * @config
+     * @var bool
+     */
+    private static $disable_current_locale_message = false;
```

Comment:
We aren't actually using this any more - can we remove it?

Reply:
Done.